cloud-common/src/main/resources/application.yml

```diff
@@ -136,7 +136,7 @@ cb:
                 CDH 6.2 - Data Science: Apache Spark, Apache Hive, Impala=cdh6-data-science;
                 CDH 6.2 - Data Lake: Hive Metastore=cdh6-shared-services;
                 CDP 1.0 - Data Mart: Apache Spark, Apache Hive, Impala, Hue=cdp-data-mart;
-                CDP 1.0 - Data Engineering: Apache Spark, Apache Livy, Apache Zeppelin, Hue=cdp-data-engineering;
+                CDP 1.0 - Data Engineering: Apache Spark, Apache Hive, Apache Oozie=cdp-data-engineering;
```

Comment:
Since you have renamed the blueprint you need to get rid of the old. You can do it in this way:

https://github.com/hortonworks/cloudbreak/blob/41d129f531b8dd2a9506b7ae1e57162eaa13384d/core/src/main/resources/schema/app/20190522115111_CB-1512_remove_hive_metastore_blueprint.sql#L4
```
UPDATE blueprint SET status = 'DEFAULT_DELETED' WHERE status = 'DEFAULT' AND name = 'CDP 1.0 - Data Engineering: Apache Spark, Apache Livy, Apache Zeppelin, Hue';
```

Reply:
Done